content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -99,6 +99,237 @@ Printer`. Follow the steps and select the *make* and *model* corresponding to th
    Epson and Star receipt printers and Zebra label printers do not need a driver to work. Make sure
    that no driver is selected for those printers.
 
+Epson special case
+~~~~~~~~~~~~~~~~~~
+
+When printing a receipt with an Epson printer, the command that is used is called `GS v 0`. However,
+there are some Epson printers models that do not support this command, they include the following
+models:
+
+- TM-U220
+- TM-U230
+- TM-P60
+- TMP-P60II
+
+If the :abbr:`POS (Point of Sale)` system is using one of these printer models, a particular
+process must be undertaken in order to use another command; `ESC *`.
+
+Process to force ESC * command
+******************************
+
+The first step is to check whether the printer is incompatible with `GS v 0` command.
+
+.. seealso::
+   See this `documentation on Epison's site
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=94>`_ for compatible
+   printers.
+
+Should the printer not be compatible with the `GS v 0` command ensure that it is
+compatible with the `ESC *` command.
+
+.. seealso::
+   See the `Epson documentation
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=88>`_ for compatible
```

Comment:
```suggestion
   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=88>`_ for `ESC *` compatible
```

Reply:
the "see also" above mention "documentation on Epison's site" while here we write "Epson documentation", maybe we should be more consistent ?
Could be a good occasion to merge both ideas:
` See the "Epson ESC* documentation" <https://..."` for this one and ` See the "Epson GS V 0 documentation" <https://..."` for the first ?

Reply:
@lse-odoo 

We get the following build error when the reference text is the same: 
`
/home/tiku/github/odoo/documentation/content/applications/productivity/iot/config/troubleshooting.rst:3: WARNING: Duplicate explicit target name: "documentation on epison's site".`

Reply:
I spoke with our Web Designer (@samueljlieber), and we came to the conclusion that making a small difference in the text would be helpful so we don't get any build errors in the long-run.